Sources/Commands/SwiftTool.swift

```diff
@@ -397,9 +398,15 @@ public class SwiftTool<Options: ToolOptions> {
                     to: sigaction.__Unnamed_union___sigaction_handler.self)
               #endif
                 sigaction(SIGINT, &action, nil)
+              #endif
 
+              #if os(Windows)
+                // Exit as if by signal()
+                TerminateProcess(GetCurrentProcess(), 3)
```

Comment:
[Apparently](https://docs.microsoft.com/en-us/previous-versions/xdkz3x12(v%3Dvs.140)), by default, on an uncaught `signal`, "By default, signal terminates the calling program with exit code 3, regardless of the value of sig."